landoapi/api/landings.py

```diff
@@ -118,6 +127,21 @@ def update(landing_id, data):
         revision (sha) of push if landed == true
         empty string if landed == false
     """
+    if not hmac.compare_digest(request.headers['API-Key'], TRANSPLANT_API_KEY):
```

Comment:
this will throw a `KeyError` exception if the `API-Key` header is missing.  treat a missing `API-Key` header the same as an incorrect api-key.